packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -240,10 +235,14 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
       ),
     );
   }
+
+  void refresh() {
+    setState(() {});
```

Comment:
it would be nice to avoid this refresh method and instead add methods to this class that modify the state as well as calling setState.

Reply:
done